src/main/java/io/fabric8/elasticsearch/plugin/kibana/KibanaSeed.java

```diff
@@ -407,10 +387,21 @@ private static void executeDelete(String index, String type, String id, Client e
 		
 		logger.debug("DELETE: '{}/{}/{}'", index, type, id);
 		
+		int timeout = 30,
+				count = 0;
+		
 		DeleteRequest request = esClient.prepareDelete(index, type, id).request();
 		try {
-			esClient.delete(request).get();
-		} catch (InterruptedException | ExecutionException e) {
+			ActionFuture<DeleteResponse> response = esClient.delete(request);
+			// wait until response isDone()
+			while ( !response.isDone() ) {
```

Comment:
Updating the try-catch to be
```
try {
  DeleteResponse response = esClient.prepareDelete(...).get();
catch { 
...
}
```

gets us this stack-trace:
```
java.lang.NullPointerException
	at org.elasticsearch.action.ActionWriteResponse.writeTo(ActionWriteResponse.java:57)
	at org.elasticsearch.action.delete.DeleteResponse.writeTo(DeleteResponse.java:101)
	at org.elasticsearch.transport.netty.NettyTransportChannel.sendResponse(NettyTransportChannel.java:97)
	at org.elasticsearch.transport.netty.NettyTransportChannel.sendResponse(NettyTransportChannel.java:75)
	at org.elasticsearch.action.support.replication.TransportReplicationAction$OperationTransportHandler$1.onResponse(TransportReplicationAction.java:248)
	at org.elasticsearch.action.support.replication.TransportReplicationAction$OperationTransportHandler$1.onResponse(TransportReplicationAction.java:244)
	at org.elasticsearch.action.support.TransportAction$ResponseFilterChain.proceed(TransportAction.java:212)
	at com.floragunn.searchguard.filter.SearchGuardFilter.apply(SearchGuardFilter.java:171)
	at org.elasticsearch.action.support.TransportAction$ResponseFilterChain.proceed(TransportAction.java:210)
	at io.fabric8.elasticsearch.plugin.KibanaUserReindexAction.apply(KibanaUserReindexAction.java:162)
	at org.elasticsearch.action.support.TransportAction$ResponseFilterChain.proceed(TransportAction.java:210)
	at org.elasticsearch.action.support.TransportAction$FilteredActionListener.onResponse(TransportAction.java:237)
	at org.elasticsearch.action.support.TransportAction$FilteredActionListener.onResponse(TransportAction.java:223)
	at org.elasticsearch.action.support.replication.TransportReplicationAction$ReroutePhase.finishOnSuccess(TransportReplicationAction.java:589)
	at org.elasticsearch.action.support.replication.TransportReplicationAction$ReroutePhase$1.handleResponse(TransportReplicationAction.java:514)
	at org.elasticsearch.action.support.replication.TransportReplicationAction$ReroutePhase$1.handleResponse(TransportReplicationAction.java:500)
	at org.elasticsearch.transport.TransportService$DirectResponseChannel.processResponse(TransportService.java:819)
	at org.elasticsearch.transport.TransportService$DirectResponseChannel.sendResponse(TransportService.java:803)
	at org.elasticsearch.transport.TransportService$DirectResponseChannel.sendResponse(TransportService.java:793)
	at org.elasticsearch.transport.DelegatingTransportChannel.sendResponse(DelegatingTransportChannel.java:58)
	at org.elasticsearch.transport.RequestHandlerRegistry$TransportChannelWrapper.sendResponse(RequestHandlerRegistry.java:134)
	at org.elasticsearch.action.support.replication.TransportReplicationAction$ReplicationPhase.doFinish(TransportReplicationAction.java:988)
	at org.elasticsearch.action.support.replication.TransportReplicationAction$ReplicationPhase.doRun(TransportReplicationAction.java:860)
	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37)
	at org.elasticsearch.action.support.replication.TransportReplicationAction$PrimaryPhase.finishAndMoveToReplication(TransportReplicationAction.java:715)
	at org.elasticsearch.action.support.replication.TransportReplicationAction$PrimaryPhase.doRun(TransportReplicationAction.java:659)
	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37)
	at org.elasticsearch.action.support.replication.TransportReplicationAction$PrimaryOperationTransportHandler.messageReceived(TransportReplicationAction.java:279)
	at org.elasticsearch.action.support.replication.TransportReplicationAction$PrimaryOperationTransportHandler.messageReceived(TransportReplicationAction.java:271)
	at com.floragunn.searchguard.ssl.transport.SearchGuardSSLTransportService.messageReceivedDecorate(SearchGuardSSLTransportService.java:161)
	at com.floragunn.searchguard.transport.SearchGuardTransportService.messageReceivedDecorate(SearchGuardTransportService.java:190)
	at com.floragunn.searchguard.ssl.transport.SearchGuardSSLTransportService$Interceptor.messageReceived(SearchGuardSSLTransportService.java:100)
	at org.elasticsearch.transport.RequestHandlerRegistry.processMessageReceived(RequestHandlerRegistry.java:75)
	at org.elasticsearch.transport.TransportService$4.doRun(TransportService.java:376)
	at org.elasticsearch.common.util.concurrent.AbstractRunnable.run(AbstractRunnable.java:37)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

I still see that the record was deleted... it just couldn't get me a proper reponse